deps/libgit2.mk

```diff
@@ -84,6 +84,11 @@ $(LIBGIT2_SRC_PATH)/libgit2-mbedtls-writer-fix.patch-applied: $(LIBGIT2_SRC_PATH
 		patch -p1 -f < $(SRCDIR)/patches/libgit2-mbedtls-writer-fix.patch
 	echo 1 > $@
 
+$(LIBGIT2_SRC_PATH)/libgit2-mbedtls-verify.patch-applied: $(LIBGIT2_SRC_PATH)/source-extracted
```

Comment:
Ok. The patch `libgit2-mbedtls-writer-fix.patch` also modifies the same file as the new patch but a separate section of the file. Since the two patches do not conflict the order which they are applied shouldn't matter so I'll just do the dependency on `libgit2-mbedtls.patch`